src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/vet/VetAddVideoComponentObject.java

```diff
@@ -124,10 +125,15 @@ public String getVideoName() {
     return this.videoName;
   }
 
-  public void verifySuggestionsIsDisplayed() {
-    wait.forElementVisible(suggestedVideo);
-    PageObjectLogging.log("verifySuggestionsIsDisplayed",
-                          "Verified suggested module appeared", true, driver);
+  public boolean areSuggestionsDisplayed() {
+    try {
+      wait.forElementVisible(suggestedVideo);
+      return true;
+    } catch(NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)